src/ui/src/core/index.ts

```diff
@@ -82,6 +82,7 @@ export function generateCore() {
 		() => Number(writerApplication.value?.organizationId) || undefined,
 	);
 	const writerAppId = computed(() => writerApplication.value?.id);
+	const writerApiKey = computed(() => "apiKey" in writerApplication.value ? writerApplication.value.apiKey : undefined);
```

Comment:
If `writerApplication.value` is undefined, you could have this error.

> Cannot use 'in' operator to search for 'apiKey' in undefined

So it's safer to do

```suggestion
	const writerApiKey = computed(() => writerApplication.value?.apiKey);
```